### PR TITLE
Reword the readme and split contributor information into another file

### PR DESCRIPTION
**Pull request by ninjabear**
#159 was the original source of this, but I remembered finding it quite hard to understand the readme before because it contains a lot of information on various aspects of nickel development, imo all it really needs to have is a status, how to helloworld and where to find out more. 

I appreciate the readme is pretty subjective and am not suggesting this PR be accepted entirely as is - I'm open to changing it up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,31 @@
...
 
-To build on nightly, you will need to add `--features nightly` to the end of the above commands. Alternatively, if depending on the library you will need to add the following to your `Cargo.toml`.
+name = "my-nickel-app"
+version = "0.0.1"
+authors = ["yourname"]
 
-```not_rust
 [dependencies.nickel]
 version = "*"
 features = ["nightly"]
```
Comment (Ryman) [inline README.md]: This is only required for nightly, and it's "unstable" now instead of "nightly" :sob:

File: README.md [hunk 2]
```diff
@@ -1,64 +1,31 @@
...
+
+[dependencies]
+rustc-serialize = "*"
```
Comment (Ryman) [inline README.md]: `rustc-serialize` was required for the old readme example before I trimmed it down, it's now only needed for some of the examples. It's probably worth adding some sort of explanation within the examples themselves (possibly comments beside the imports) or some comment similar to 'you can find the crates required to build the examples on crates.io'. Not sure which is best!

File: README.md [hunk 3]
```diff
@@ -78,20 +45,22 @@ fn main() {
...
 
-##License
+## More examples
+
+More examples can be found [in the examples directory](/examples/) and the full documentation can be [found here](http://nickel-org.github.io/nickel/).
 
-Nickel is open source and licensed with the [MIT license](https://github.com/nickel-org/nickel/blob/master/LICENSE)
+##License
 
+[nickel.rs](http://nickel.rs) is open source and licensed with the [MIT license](https://github.com/nickel-org/nickel/blob/master/LICENSE)
```
Comment (Ryman) [inline README.md]: Can probably drop most of the line pre- "[MIT](..)"

**Comment by Ryman** [comment]
This looks like a good improvement :+1: 

I've been meaning to get round to this for a while, my preference would be to cut it down even a bit more (I'm kind of [inspired by this recently](https://speakerdeck.com/rowanmanning/ux-for-your-node-modules)). I do think we should slip the example code above the installation instructions for sure!

**Comment by ninjabear** [comment]
I really like the readme proposed in that link too - its kind of where I was going with this but I think I've missed some bits out and given some things slightly odd names (Usage->hello world). I can change it to be more like that if you like @Ryman ?

**Comment by Ryman** [comment]
My preference would be to put the code sample where dependencies are right now, without any titles (its implied) and then put dependencies below as 'Installation' or similar. But other than that I think most of this PR is worth taking as-is to iterate on :+1: 

@cburgdorf any opinion on this?

**Comment by ninjabear** [comment]
@Ryman I went through your comments and implemented, think I caught them all? 

I changed "nightly" to "unstable" but I'm not sure if you think we should remove the channel altogether?

**Comment by Ryman** [comment]
Looks good, much appreciated @ninjabear!

**Comment by ninjabear** [comment]
np :)

**Pull request merged**